front/src/components/Task/TaskTagsListCondensed.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div class="row">
+    <task-tag-chip
+      v-if="first_tag !== undefined"
+      :class="{ 'tag-smaller-width': tags.length > 1 }"
+      :name="first_tag.tag"
+      :dense="dense"
+    />
+
+    <q-chip
+      v-if="tags.length > 1"
+      text-color="white"
+      color="black"
+      class="non-selectable"
+      :ripple="false"
+      :dense="dense"
+    >
+      <q-tooltip
+        anchor="center right"
+        self="center left"
+        :offset="[0, 0]"
+        class="transparent"
+        transition-show="fade"
+        transition-hide="fade"
+      >
+        <q-card bordered style="border-radius: 23px !important;">
+          <q-card-section class="tooltip-section" style="padding: 4px;">
+            <task-tag-chip
+              v-for="tag in remaining_tags"
+              :key="tag.nodeId"
+              :name="tag.tag"
+              :dense="dense"
+            />
+          </q-card-section>
+        </q-card>
+      </q-tooltip>
+
+      <span class="tag-chip">
+        +{{ tags.length - 1 }}
+      </span>
+    </q-chip>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+import { Tag } from 'src/ctfnote/models';
+import TaskTagChip from './TaskTagChip.vue';
+
+export default defineComponent({
```

Comment:
Just for my understanding: the TaskTagsListCondensed is the version with the '+1' and the TaskTagsList is the original version and you have chosen to not merge those two implementations into one file with a boolean switch to enable/disable the '+1'? Is there any reason to not do that?

Reply:
Good point. They are now merged into a single class.